core/src/main/java/hudson/security/PermissionGroup.java

```diff
@@ -110,7 +133,7 @@ public int compareTo(PermissionGroup that) {
 
         // among the permissions of the same group, just sort by their names
         // so that the sort order is consistent regardless of classloading order.
-        return id().compareTo(that.id());
+        return getOwnerClassName().compareTo(that.getOwnerClassName());
```

Comment:
Why these changes from `id()` to `getOwnerClassName()`?

Reply:
Nevermind: `id()` used to be the owner class, and is now the English title. So this just retains previous behavior.

Reply:
Yes